RxSwiftWorkshop/Maps/ViewModel/FlightVisualizationViewModel.swift

```diff
@@ -0,0 +1,60 @@
+//
+// Created by Krzysztof Siejkowski on 03/05/2017.
+// Copyright (c) 2017 Mobile Academy. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+import CoreLocation
+
+protocol FlightVisualisation {
+    var route: Observable<Route> { get }
```

Comment:
I wonder if it would be better to have it as a `Variable<Route>`. It seems to me more intuitive.
Later on, you can do `asObservable` and subscribe to changes anyways.
Drawback is, that you can change `Variable` from the outside.